lib/runner.js

```diff
@@ -289,19 +293,20 @@ class Runner extends Emittery {
 			powerAssert: this.powerAssert,
 			title: `${task.title}${titleSuffix || ''}`,
 			isHook: true,
-			testPassed
+			testPassed,
+			emitStateChange: this.emitStateChange
```

Comment:
I wouldn't pass the entire `emitStateChange`. How about a `notifyOfTimeout()` function that calls `emitStateChange()` internally?

Reply:
created a bespoke function for timeout: `notifyTimeoutUpdate()`